ortools/linear_solver/xpress_interface_test.cc

```diff
@@ -766,6 +766,14 @@ TEST_F(XpressFixtureLP, SetPrimalTolerance) {
   EXPECT_EQ(getter.getDoubleControl(XPRS_FEASTOL), tol);
 }
 
+TEST_F(XpressFixtureLP, SetPrimalToleranceNotOverridenByMPSolverParameters) {
+  double tol = 1e-4;  // Choose a value different from kDefaultPrimalTolerance
+  std::string xpressParamString = "FEASTOL " + std::to_string(tol);
+  solver.SetSolverSpecificParametersAsString(xpressParamString);
+  solver.Solve();
+  EXPECT_EQ(getter.getDoubleControl(XPRS_FEASTOL), tol);
+}
```

Comment:
Add tests for other types of "controls" ? I see 4 in total
- String
- Double (OK)
- Integer
- Integer64

Reply:
The goal here is to test that parameters available as MPSolverParameters do not override the corresponding parameter set from specific parameters. And all relevant MPSolverParameters are tested